.github/workflows/autopr.yaml

```diff
@@ -0,0 +1,44 @@
+name: Manual workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      majorVersion:
+        description: 'Major Version e.g. 3.0'
+        required: true
+      targetVersion:
+        description: 'Target Version e.g. 3.0.151471'
+        required: true
+
+jobs:
+  createPR:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: dev
+    - id: createBranch
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
```

Comment:
We have functions git hub user : https://github.com/azfuncgh :)

Lets use same user across repos. Reach out to  @yojagad  if needed , on how to use this user. azfuncgh is used in host and webjobs sdk repos for automated PRs

Reply:
Oh, I didn't know that! Hi @yoalgh  I'd like to know the e-mail account of azfuncgh !

Reply:
Fixed Yogesh's alias in above comment.

Reply:
It's azfuncgh@github.com . If you need PAT for that, please ping @fabiocav for the same :)

Reply:
thanks. Will follow up with @fabiocav offline.